compassion_nordic_accounting/models/account_payment_order.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields
+
+
+class AccountPaymentOrder(models.Model):
+    _inherit = 'account.payment.order'
+
+    skip_payment_line = fields.Boolean(string="Skip payment lines",
+                                       help="Should the payment lines be generated ?",
```

Comment:
The help will introduce confusion for the user. Either rename the field to `generate_payment_lines` and reverse the setting, or remove the help message.
```suggestion
```